docs/reference/experimental/sync/agent.md

```diff
@@ -0,0 +1,34 @@
+# Agent
+
+## Example Script:
+
+<details class="quote">
+  <summary>Working with a Synapse Agent</summary>
+
+```python
+{!docs/scripts/object_orientated_programming_poc/oop_poc_agent.py!}
+```
+</details>
+
+## API Reference
+
+::: synapseclient.models.Agent
+    options:
+        members:
+        - register
+        - get
+        - start_session
+        - get_session
+        - prompt
```

Comment:
These functions are not showing on the doc site:
https://synapsepythonclient--1152.org.readthedocs.build/en/1152/reference/experimental/sync/agent/#synapseclient.models.Agent-functions

I think because it's missing `inherited_members: true`

Reply:
I got the functions to show up for `Agent`, but they aren't showing up for `Activitiy`, `Team`, or `UserProfile` in the synchronous section and adding `inherited_members: true` didn't work 🤔

Reply:
@BryanFauble It looks like those aren't showing up in the current version of the [docs](https://python-docs.synapse.org/reference/oop/models/) too. Was there a reason why that wasn't happening before that I'm not aware of?

Reply:
> Was there a reason why that wasn't happening before that I'm not aware of?

Nope. Just that I missed it. What you have currently looks great!